tests/helpers/WebView.ts

```diff
@@ -45,13 +47,13 @@ class WebView {
     async switchToContext (context:string) {
         // The first context will always be the NATIVE_APP,
         // the second one will always be the WebdriverIO web page
-        await driver.switchContext((await this.getCurrentContexts())[context === CONTEXT_REF.NATIVE ? 0 : 1]);
+        await driver.switchContext(String(await this.getCurrentContexts())[context === CONTEXT_REF.NATIVE ? 0 : 1]);
```

Comment:
Why casting to `String` here?

Reply:
Typing requires a string

Reply:
I know that but what does `(await this.getCurrentContexts())[context === CONTEXT_REF.NATIVE ? 0 : 1]` return that requires to cast it?

Reply:
a Context

Reply:
So what you are doing is to cast `DetailedContext` into a string? I don't think that makes sense. Is there a chance to remove the casting, as it seems wrong to me. Please clarify if I am mistaken here. Thanks!

Reply:
There is another option which is to remove the import line then to cast `Context[]` to `string[]` using `as`, otherwise, you'll have to take this cast, ex: https://github.com/pass-culture/pass-culture-app-native/pull/3876/files#diff-194b8610d4c3c20f4489800819e1ef63a788eff06c088631ed0e822b3ba35168R28

I can give a proper error later on tomorrow.

Reply:
See answer below